app/js/models/addon_generator.js

```diff
@@ -147,50 +148,107 @@ el.${name} = ${JSON.stringify(value)};
     }
   }
 
-  opMoveAppend(target) {
+  opCopyAppend(destination) {
+    this.operations.push(
+`/*=AddonGenerator::copyAppend*/
+var destination = document.querySelector('${getSelector(destination)}');
+var template = document.createElement('template');
+template.innerHTML = \`${this.target.outerHTML.replace(/\`/g, '\\\`')}\`;
+if (destination) {
+  destination.appendChild(template.content);
+}
+/*==*/`
+    );
+  }
+
+  opCopyPrepend(destination) {
+    this.operations.push(
+`/*=AddonGenerator::copyPrepend*/
```

Comment:
There's a lot of duplicated code between each of these operations. They definitely seem like good candidates for some refactoring.